### ISSUE TITLE
Wraith 0.3 / sending live snaps for indefinitely  and can't complete

### ISSUE THREAD

**Issue by bartekx32** [AeonLucid/SnapHide#7]
**Device**: iPhone 8
**Jailbreak**: unc0ver 
**Snapchat version**: 10.55.1

**Describe the bug**
It's impossible to send live snaps because it indefinitely keeps sending the snap.

**Additional context**
Hope you will fix it in the very future.

**Comment by AeonLucid** [comment]
Does it work with Wraith uninstalled?

**Comment by bartekx32** [comment]
no still can't send.

**Comment by bartekx32** [comment]
> Does it work with Wraith uninstalled?

no still can't send.

**Comment by AeonLucid** [comment]
Are you talking about sending a "normal snap"? Can you explain your steps?

**Comment by bartekx32** [comment]
> Are you talking about sending a "normal snap"? Can you explain your steps?

Yes, when I take a photo and send it to anyone (even myself) it keeps indefinitely sending it. When the screen pops back up to the screen with my friends list it says "Sending..." under their name and the little circle keeps spinning around like when you send a snap or video to someone.

**Comment by bartekx32** [comment]
When I uninstall the tweak it goes back to normal, can send snaps again.

**Comment by AeonLucid** [comment]
I can't seem to reproduce this on the exact same setup, do you have any other tweaks installed?

**Comment by bartekx32** [comment]
App Admin, Filza file manager.

**Comment by bartekx32** [comment]
uinstalled those two aps and still. Have you tried it yourself?

**Comment by AeonLucid** [comment]
Yes it worked fine for me with SnapHide installed. Have you tried all versions of SnapHide?

**Comment by bartekx32** [comment]
> Yes it worked fine for me with SnapHide installed. Have you tried all versions of SnapHide?

Only 1.2, shall I try other versions?

it doesnt let me send snaps on all versions as i can see

**Comment by bartekx32** [comment]
> Yes it worked fine for me with SnapHide installed. Have you tried all versions of SnapHide?

Heres whole list of my apps installed in Cydia:

System Memory Reset
Substrate Safe Mode
Snap Hide
MobileSubstrate Dummy
Jailbreak Resources
Iphone Firmware /sbin
Cydia Translations
Cydia Substrate
Cydia Installer
BigBoss Icon Set
AppList

**Comment by bartekx32** [comment]
Do i have to turn it on somewhere? or just install and thats it?

**Comment by GODLIKE5** [comment]
For the mentioned issue above I did find a bypass.. what you have to do is turn on a VPN then respring. After the respring open Snapchat again and turn of your vpn. Tested on latest wraith and Snapchat 10.65 ( iPhone 8+ 12.4 )

Edit: after a  random respring or app close this would not work and it is something you have to keep doing. If not wait about 1-2 minutes until it delivers

**Comment by bartekx32** [comment]
> For the mentioned issue above I did find a bypass.. what you have to do is turn on a VPN then respring. After the respring open Snapchat again and turn of your vpn. Tested on latest wraith and Snapchat 10.65 ( iPhone 8+ 12.4 )
> 
> Edit: after a random respring or app close this would not work and it is something you have to keep doing. If not wait about 1-2 minutes until it delivers

how do i respring? im new to ios

**Comment by bartekx32** [comment]
> For the mentioned issue above I did find a bypass.. what you have to do is turn on a VPN then respring. After the respring open Snapchat again and turn of your vpn. Tested on latest wraith and Snapchat 10.65 ( iPhone 8+ 12.4 )
> 
> Edit: after a random respring or app close this would not work and it is something you have to keep doing. If not wait about 1-2 minutes until it delivers

Doesnt work did same thing as u said

**Comment by GODLIKE5** [comment]
> how do i respring? im new to ios

I recommend pull2respring. Then basically go to settings and pull all the way down. Also for everyone here wondering what's up, Snapchat servers are currently bugging so messages might not send or take forever to send. Many of the people I have on my Snapchat are saying it's glitching and they're on latest stock os so problem's on snapchat servers not tweak related as far as I know.
